server/Data/Game.cs

```diff
@@ -0,0 +1,22 @@
+namespace server.Data
+{
+
+    public class Game
+    {
+        
+        public int GameId { get; set; }
+        public int Season { get; set; }
+        public int SeasonType { get; set; }
+        public DateTime GameDate { get; set; }
+        public DateTime GameDateTime { get; set; }
+        public int HomeScore { get; set; }
+        public int AwayScore { get; set; }
+        public virtual Team HomeTeam { get; set; } = null!;
+        public int HomeTeamId { get; set; }
+        public virtual Team AwayTeam { get; set; } = null!;
+        public int AwayTeamId { get; set; }
+        //public virtual ICollection<PlayByPlay> PlayByPlays { get; set; } = null!;
```

Comment:
needed?

Reply:
Not necessarily needed but its a TODO item. It is mentioned in AppDbContext as todo but nowhere else. We can remove the commented out stuff if you'd prefer. I figured since we may alter the schema as we go if we decide certain attributes belong to different tables, that it might be beneficial to keep them there until its finalized. I'm fine either way though.